scss/_facets.scss

```diff
@@ -0,0 +1,100 @@
+// The facet wrapper (`<div>`)
+.facet {
+  position: relative;
+}
+
+.facet-toggle {
+  display: flex;
+
+  // Generate the caret automatically
+  &:after {
+    display: inline-block;
+    width: 0;
+    height: 0;
+    margin-left: auto;
+    vertical-align: middle;
+    content: "";
+    border-top: $caret-width solid;
+    border-right: $caret-width solid transparent;
+    border-left: $caret-width solid transparent;
+    align-self: center;
+
+    -webkit-transition: transform 0.5s; /* Safari */
+    transition: transform 0.5s;
+    -ms-transform: rotate(180deg); /* IE 9 */
+    -webkit-transform: rotate(180deg); /* Chrome, Safari, Opera */
+    transform: rotate(180deg);
+  }
+
+  &.collapsed:after {
+    -ms-transform: rotate(0deg); /* IE 9 */
+    -webkit-transform: rotate(0deg); /* Chrome, Safari, Opera */
+    transform: rotate(0deg);
+  }
+
+  // Prevent the focus on the facet toggle when closing facets
+  &:focus {
+    outline: 0;
+  }
+}
+
+// Links, buttons, and more within the facet menu
+//
+// `<button>`-specific styles are denoted with `// For <button>s`
```

Comment:
I'm guessing a lot of these comments are copied from bootstrap? Let's stick with normal commenting convention - finishing with a full stop.